openwisp_users/static/openwisp-users/js/uuid.js

```diff
@@ -0,0 +1,12 @@
+/*global django */
+django.jQuery(function ($) {
+  "use strict";
+  $('.field-id label').text('UUID:');
+  var container = $('.field-id .readonly'),
+    value = container.text();
+  container.html('<input readonly id="id_id" type="text" class="vTextField readonly" value="' + value + '">');
+  var id = $('#id_id');
+  id.click(function () {
+    $(this).select();
+  });
+});
```

Comment:
replace this code with the following:

```javascript
django.jQuery(function($) {
    var container = $('.field-id_str .readonly').eq(0),
        value = container.text();
    container.html('<input readonly id="id_id" type="text" class="vTextField readonly" value="'+ value +'">');
    var id = $('#id_id');
    id.click(function(){
        $(this).select();
    });
});
```

Reply:
In `openwisp-users` is `jslint` which requires 2 space instead of tab

Reply:
and `jslint` requires to add `use strict` otherwise travis will return error

Reply:
And I think that there is no need to add `eq(0)` (if there is any reason for that please correct me)

Reply:
improve it as needed, no problem